src/components/forums/topic/CreatePost.tsx

```diff
@@ -72,10 +85,14 @@ export function CreatePost(props: CreatePostProps) {
         ),
         type: MessageType.success,
       });
+
       setTimeout(
         () => setIsNotificationHidden(true),
         NOTIFICATION_BANNER_TIMEOUT
-      );
+        );
+      if (tx) {
+        await Forum.connection.confirmTransaction(tx).then(() => update());
```

Comment:
for future PR... can we make this update only for certain aspects of the forum? like updatePosts, updateTopics, etc.

Reply:
Yes, this is definitely doable. I wrote a draft of that earlier but it was unstable so I shelved it. Can re-add when I have a moment